app/models/course.rb

```diff
@@ -41,6 +41,7 @@ class Course < ApplicationRecord
   has_one_attached :thumbnail
   has_one_attached :cover
 
+  default_scope { order(sort_index: :asc) }
```

Comment:
Why do we need this?
It is best practice to avoid using `default_scope`. If we need this in certain places let's be explicit and do the sorting inline not globally.